### PR TITLE
4 article list

### PR DESCRIPTION
**Pull request by mejaz4**

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,24 +1,26 @@
...
-        >
-          Learn React
-        </a>
-      </header>
-    </div>
+    <div className="app">
+    <Header />
+    <Navbar />
+
+    <div className="main">
```
Comment (leekli) [inline src/App.js:14]: Maybe a semantic HTML tag such as <main> would be more appropiate here?

File: src/App.js [hunk 2]
```diff
@@ -1,24 +1,26 @@
...
+        <Routes>
+          <Route path="/" element={<Articles />}></Route>
+          <Route path="/topics" element={<Topics />}></Route>
```
Comment (leekli) [inline src/App.js:17]: Good use of routes! Well done 😎

File: src/Articles.jsx [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+        getArticles()
+        .then((data) => {
+            setArticleList(data)
+            setIsLoading(false)
+        })
+    },[])
+
+   if(isLoading) {
+    return <p className='Loading'>Loading...</p>;
+   }
```
Comment (leekli) [inline src/Articles.jsx:20]: Good to see a loading design pattern, good work!

File: src/Articles.jsx [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+
+
+    return (
+        <div className="article-list">
```
Comment (leekli) [inline src/Articles.jsx:24]: Is there a more appropriate semantic HTML tag which would work here? Take a look at: [https://www.w3schools.com/html/html5_semantic_elements.asp](url)

File: src/api.js [hunk 5]
```diff
@@ -0,0 +1,22 @@
...
+    baseURL: 'https://musasnews.onrender.com/api'
+    
+})
+
+export const getArticles = () => {
+    return myApi.get('/articles')
+    .then((res) => {
+        return res.data.articles
+    })
+}
```
Comment (leekli) [inline src/api.js:14]: Good to see you've extracted the api call functionality into its own file 👍

**Review by leekli — APPROVED** [review]
Hey Musa, well done on this ticket!

I've left a couple of comments for you however i'm happy for you to merge this branch.

The comments are around the use of proper semantic HTML tags and I'd encourage you to use more appropiate ones, instead of div's both in this ticket and future ones.

Well done so far :D

File: src/App.js (hunk 1 above)
Comment (mejaz4) [reply src/App.js:14]: You mean <main> tag here?

Comment (mejaz4) [reply src/App.js:14]: main tag

File: src/App.js (hunk 2 above)
Comment (mejaz4) [reply src/App.js:17]: thanks!

**Pull request merged**